testing/cockroach/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Joe Holden <jwh@zorins.us>
+# Maintainer: Joe Holden <jwh@zorins.us>
+pkgname=cockroach
+pkgver=2.1.6
+pkgrel=0
+pkgdesc="The open source, cloud-native SQL database."
+url="https://cockroachlabs.com"
+arch="x86_64"
+license="Apache-2.0"
+makedepends="go bash cmake autoconf linux-headers ncurses-dev"
+install="$pkgname.pre-install"
+pkgusers="$pkgname"
```

Comment:
I'm not sure if it makes sense to let this depend on the package name. If the package would be renamed, or someone creates a specialized version of the package, it should probably still use the same user/group.

As the user/group are static in the pre-install script anyway, it makes sense to make them static here as well.w

Reply:
Updated in my last push